lib/centralisation/providers/showLinkDetails.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:myecl/centralisation/class/module.dart';
+import 'package:flutter/material.dart';
+import 'package:myecl/centralisation/providers/openLink.dart';
+
+void showLinkDetails(BuildContext context, Module module) {
+  showDialog(
+    context: context,
+    builder: (BuildContext context) {
+      return AlertDialog(
+        title: Text(module.name),
+        content: Text(module.description),
+        actions: [
+          TextButton(
+            child: Text('Accéder au site'),
```

Comment:
You can create text constant to make every text editable without searching for it in every file